panther_analysis_tool/main.py

```diff
@@ -88,6 +88,7 @@
 RULE = "rule"
 
 RESERVED_FUNCTIONS = [
+    "alert_context",
```

Comment:
Just to note that except for the addition of the function name here, no other change should be necessary with the changes in https://github.com/panther-labs/panther_analysis_tool/pull/109. No problem of course with whichever gets merged first.